benchmark/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
```

Comment:
Should add `Dojo` as well?

Reply:
You probably want to `dev ..` in the Pkg mode.

Reply:
Surprisingly, `Dojo` doesn't need to be listed as a dependency. The issue was that BenchmarkCI by default compares to the "master" branch and not "main" branch.